modules/rpc/handler.go

```diff
@@ -26,6 +26,7 @@ import (
 	"go.uber.org/fx"
 	"go.uber.org/fx/internal/fxcontext"
 	"go.uber.org/fx/service"
+	"go.uber.org/fx/uauth"
```

Comment:
Yeah, you are right. I was hoping for uber specific authentication to be plugged in eventually, may be open sourced, but `auth` seems pretty generic as well. I'll get a rename PR once this change gets in. I don't want to add unrelated stuff to this PR.